config.py

```diff
@@ -140,6 +140,29 @@
         hidden_category = 'Hidden_categories',
     ),
 
+    ar = dict(
+        lang_name = 'العربية',
+        lang_dir = 'rtl',
+        database = 'arwiki_p',
+        wikipedia_domain = 'ar.wikipedia.org',
+
+        beginners_link = 'https://ar.wikipedia.org/wiki/%D9%88%D9%8A%D9%83%D9%8A%D8%A8%D9%8A%D8%AF%D9%8A%D8%A7:%D8%A7%D9%84%D8%A7%D8%B3%D8%AA%D8%B4%D9%87%D8%A7%D8%AF_%D8%A8%D9%85%D8%B5%D8%A7%D8%AF%D8%B1',
+        beginners_link_title = 'ويكيبيديا:الاستشهاد بمصادر',
+
+        reliable_sources_link = 'https://ar.wikipedia.org/wiki/%D9%88%D9%8A%D9%83%D9%8A%D8%A8%D9%8A%D8%AF%D9%8A%D8%A7:%D9%85%D8%B5%D8%A7%D8%AF%D8%B1_%D9%85%D9%88%D8%AB%D9%88%D9%82_%D8%A8%D9%87%D8%A7',
+
+        lead_section_policy_link = '',
+        lead_section_policy_link_title = '',
+
+        citation_needed_category = 'مقالات ذات عبارات بحاجة لمصادر',
+
+        # The name of the category for hidden categories, without the
+        # 'Category:' prefix and with underscores instead of spaces.
+        # Categories belonging to this category are typically used for
+        # maintenance and will not show up on Citation Hunt.
+        hidden_category = 'تصنيفات مخفية',
+    ),
```

Comment:
I think you're missing `citation_needed_templates`, which tells us which templates to look for in the articles (https://github.com/eggpi/citationhunt/blob/master/config.py#L73).

Typically these are the templates that are tracked by `citation_needed_category`, so for example in enwiki we use:

```python
citation_needed_category = 'All_articles_with_unsourced_statements',
citation_needed_templates = ['Citation needed']
```

Reply:
Done.